### PR TITLE
Create gateway

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -556,6 +556,41 @@ functions:
...
 gateways:
+  create-modal:
+    title: Create Gateway
+    advanced:
+      add-server: Add server
+      delete-server: Delete server
+      enable-tls: Enable TLS
+      hosts: Hosts
+      placeholders:
+        hosts: 'Enter the hosts #e.g. *.api.mydomain.com'
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:567]: Should the '#' be there?

**Review by Wawrzyn321 — COMMENTED** [review]
When I select `TLS Mode` to "simple" and change the `Credential Name` input, `TLS Mode` value switches back to "Select TLS Mode". And vice versa, when I choose mode credentials disappear.

**Comment by Wawrzyn321** [comment]
Would it be possible to force the user to fill TLS form when HTTPS port is used? Now when I choose HTTPS and ignore TLS the form will be valid, but the request will fail.

**Comment by Wawrzyn321** [comment]
Changes from `Hosts` field (simple form) or from YAML aren't propagated to `Hosts` in advanced form.

**Review by alexandra-simeonova — DISMISSED** [review]

**Review by Sawthis — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**